docs/wiki/CustomerUsage.md

```diff
@@ -24,27 +24,30 @@ module modCustomerUsageAttribution '../../CRML/customerUsageAttribution/cuaIdTen
-| getManagementGroupName         | cff0ca56-5d8c-4594-bf79-5c046809b017 |
-| hubNetworking                  | 2686e846-5fdc-4d4f-b533-16dcb09d6e6c |
-| logging                        | f8087c67-cc41-46b2-994d-66e4b661860d |
-| managementGroups               | 9b7965a0-d77c-41d6-85ef-ec3dfea4845b |
-| policy-definitions             | 2b136786-9881-412e-84ba-f4c2822e1ac9 |
-| policy-assignments             | 78001e36-9738-429c-a343-45cc84e8a527 |
-| alzDefaultPolicyAssignments    | 98cef979-5a6b-403b-83c7-10c8f04ac9a2 |
-| publicIp                       | 3f85b84c-6bad-4c42-86bf-11c233241c22 |
-| resourceGroup                  | b6718c54-b49e-4748-a466-88e3d7c789c8 |
-| roleAssignments                | 59c2ac61-cd36-413b-b999-86a3e0d958fb |
-| spokeNetworking                | 0c428583-f2a1-4448-975c-2d6262fd193a |
-| subscriptionPlacement          | 3dfa9e81-f0cf-4b25-858e-167937fd380b |
-| virtualNetworkPeer             | ab8e3b12-b0fa-40aa-8630-e3f7699e2142 |
-| vwanConnectivity               | 7f94f23b-7a59-4a5c-9a8d-2a253a566f61 |
-| vnetPeeringVwan                | 7b5e6db2-1e8c-4b01-8eee-e1830073a63d |
-| privateDnsZones                | 981733dd-3195-4fda-a4ee-605ab959edb6 |
-| hubSpoke - Orchestration       | 50ad3b1a-f72c-4de4-8293-8a6399991beb |
-| hubPeeredSpoke - Orchestration | 8ea6f19a-d698-4c00-9afb-5c92d4766fd2 |
+
+The following are the unique ID's (also known as PIDs) used in each of the modules:
+
+| Module Name                     | PID                                  |
+| ------------------------------- | ------------------------------------ |
+| customRoleDefinitions           | 032d0904-3d50-45ef-a6c1-baa9d82e23ff |
+| getManagementGroupName          | cff0ca56-5d8c-4594-bf79-5c046809b017 |
+| hubNetworking                   | 2686e846-5fdc-4d4f-b533-16dcb09d6e6c |
+| logging                         | f8087c67-cc41-46b2-994d-66e4b661860d |
+| managementGroups                | 9b7965a0-d77c-41d6-85ef-ec3dfea4845b |
+| policy-definitions              | 2b136786-9881-412e-84ba-f4c2822e1ac9 |
+| policy-assignments              | 78001e36-9738-429c-a343-45cc84e8a527 |
+| alzDefaultPolicyAssignments     | 98cef979-5a6b-403b-83c7-10c8f04ac9a2 |
+| publicIp                        | 3f85b84c-6bad-4c42-86bf-11c233241c22 |
+| resourceGroup                   | b6718c54-b49e-4748-a466-88e3d7c789c8 |
+| roleAssignments                 | 59c2ac61-cd36-413b-b999-86a3e0d958fb |
+| spokeNetworking                 | 0c428583-f2a1-4448-975c-2d6262fd193a |
+| subscriptionPlacement           | 3dfa9e81-f0cf-4b25-858e-167937fd380b |
+| virtualNetworkPeer              | ab8e3b12-b0fa-40aa-8630-e3f7699e2142 |
+| vwanConnectivity                | 7f94f23b-7a59-4a5c-9a8d-2a253a566f61 |
+| vnetPeeringVwan                 | 7b5e6db2-1e8c-4b01-8eee-e1830073a63d |
+| privateDnsZones                 | 981733dd-3195-4fda-a4ee-605ab959edb6 |
+| hubSpoke - Orchestration        | 50ad3b1a-f72c-4de4-8293-8a6399991beb |
```

Comment:
Not directly related to this PR, but this module seems to be mssing.

Reply:
Nope its there - https://github.com/Azure/ALZ-Bicep/pull/298/files#diff-fe19a3b373a175486c649ce7646942d22c0b7303568cc4cf5c75a7dea24fa39cR53
![image](https://user-images.githubusercontent.com/41163455/182139325-419c1786-0201-4941-9746-5e2e2d7c4e99.png)

Reply:
I meant the one in line 51 :-) the hubSpoke